From bunk@stusta.de Tue Apr 18 15:10:34 2006
Date: Wed, 19 Apr 2006 00:10:26 +0200
From: Adrian Bunk <bunk@stusta.de>
To: Andrew Morton <akpm@osdl.org>, Kristen Accardi <kristen.c.accardi@intel.com>
Subject: PCI Hotplug: make acpiphp_glue.c:handle_hotplug_event_func() static
Message-ID: <20060418221026.GV11582@stusta.de>
Content-Disposition: inline

On Tue, Apr 18, 2006 at 03:14:23AM -0700, Andrew Morton wrote:
>...
> Changes since 2.6.17-rc1-mm2:
>...
> +acpiphp-use-new-dock-driver.patch
>...
>  ACPI fixes and features
>...

handle_hotplug_event_func() can now become static.

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Cc: Kristen Accardi <kristen.c.accardi@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/pci/hotplug/acpiphp.h      |    1 -
 drivers/pci/hotplug/acpiphp_glue.c |    5 ++++-
 2 files changed, 4 insertions(+), 2 deletions(-)

--- gregkh-2.6.orig/drivers/pci/hotplug/acpiphp.h
+++ gregkh-2.6/drivers/pci/hotplug/acpiphp.h
@@ -226,7 +226,6 @@ extern int acpiphp_glue_init (void);
 extern void acpiphp_glue_exit (void);
 extern int acpiphp_get_num_slots (void);
 typedef int (*acpiphp_callback)(struct acpiphp_slot *slot, void *data);
-void handle_hotplug_event_func(acpi_handle, u32, void*);
 
 extern int acpiphp_enable_slot (struct acpiphp_slot *slot);
 extern int acpiphp_disable_slot (struct acpiphp_slot *slot);
--- gregkh-2.6.orig/drivers/pci/hotplug/acpiphp_glue.c
+++ gregkh-2.6/drivers/pci/hotplug/acpiphp_glue.c
@@ -59,6 +59,8 @@ static LIST_HEAD(bridge_list);
 static void handle_hotplug_event_bridge (acpi_handle, u32, void *);
 static void acpiphp_sanitize_bus(struct pci_bus *bus);
 static void acpiphp_set_hpp_values(acpi_handle handle, struct pci_bus *bus);
+static void handle_hotplug_event_func(acpi_handle handle, u32 type,
+				      void *context);
 
 
 /*
@@ -1432,7 +1434,8 @@ static void handle_hotplug_event_bridge(
  * handles ACPI event notification on slots
  *
  */
-void handle_hotplug_event_func(acpi_handle handle, u32 type, void *context)
+static void handle_hotplug_event_func(acpi_handle handle, u32 type,
+				      void *context)
 {
 	struct acpiphp_func *func;
 	char objname[64];
